File: JDA/Java/noShards.java
String url = "https://discordbots.group/api/bot/"+jda.getSelfUser().getId();
String token = "";

JSONObject data = new JSONObject();
data.put("count", jda.getGuilds().size());

RequestBody body = RequestBody.create(MediaType.parse("application/json"), data.toString());

Request request = new Request.Builder()
        .url(url)
        .post(body)
        .addHeader("User-Agent", "DiscordBot " + jda.getSelfUser().getName())
        .addHeader("Authorization", token)
        .build();

try {
    new OkHttpClient().newCall(request).execute();
} catch (IOException e) {
    e.printStackTrace();
}

File: JDA/Kotlin/noShards.kt
val url = "https://discordbots.group/api/bot/${jda.selfuser.id}"
val client = OkHTTPClient()
val data : JSONObject = JSONObject()
data.put("count", jda.guilds.size())
val body : RequestBody = RequestBody.create(MediaType.parse("application/x-www-form-urlencoded"), data.toString());

val request : Request = RequestBuilder()
    .url(url)
    .post(body)
    .addHeader("Authorization", bot_key_here)
    .build()
try {
    client().newCall(request).execute()
} catch (e: Exception) {
    e.printStackTrace();
}